web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js

```diff
@@ -7,12 +7,17 @@ odoo.define('web.web_ir_actions_act_window_message', function(require)
 
     var ActionManager = require('web.ActionManager'),
         core = require('web.core'),
-        Model = require('web.Model'),
         Dialog = require('web.Dialog');
 
     var _t = core._t;
 
     ActionManager.include({
+        _handleAction: function (action, options) {
+            if (action.type === 'ir.actions.act_window.message') {
+                return this.ir_actions_act_window_message(action, options);
```

Comment:
you could consider renaming this to `_execute...` to be in line with core

Reply:
Done!